Assets/Scripts/MVC/Tank/TankController.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class TankController : MonoBehaviour
+{
+    public FixedJoystick joystick;
+  //  private float horizontalMove;
+   // private float verticalMove;
+
+    public float speed;
+    public float rotate;
+
+
+    private void Update()
+    {
+        TankMovement();
+        TankRotate();
+    }
+
+    private void TankMovement()
+    {
+        float vertical = joystick.Vertical;
+        if (vertical > .3f || vertical < -.3f)
```

Comment:
Avoid magic numbers and magic strings. Create a variable called verticalRangeMin/Max and assign the .3f/-3.f value to it and then use it in this line